### DISCUSSION TITLE
tlsfuzzer discord server

### DISCUSSION THREAD

**Discussion by JegadeeshVontlin** [tlsfuzzer/tlsfuzzer#864] [Ideas]
A dedicated discord server might be helpful for newbies to have discussions to develop the tls fuzzer.

**Comment by tomato42** [comment]
On the other hand, discussions in github are searchable through google, so while not as immediate, I think this forum is better. Especially given that the community of users is fairly small.

**Reply by JegadeeshVontlin** [reply]
agreed @tomato42

**Reply by JegadeeshVontlin** [reply]
Still, I feel it would be better to have discord channel for quick OTG discussions if you're willing or approve for it @tomato42

**Reply by tomato42** [reply]
Github automatically updates the page when a comment is posted, so when people are online, you can have a near-realitime conversation, no need for discord for that.

And especially when I'm working, I have a github page open and visible like 50% of the time. I don't use discord for work, so chance of a realtime conversation over discord would be rather minimal.
